runtime/runtime.go

```diff
@@ -18,6 +18,8 @@ import (
 // required to execute a flux script.
 var Default = &runtime{}
 
+const panicNotFinalized = "builtins not finalized (you may be missing an `import _ \"github.com/influxdata/idpe/fluxinit/static\"` statement)"
```

Comment:
I like the change, which package you import depends on which product you are developing in. Maybe change this from 

```
import _ \"github.com/influxdata/idpe/fluxinit/static\"
```

to

```
import _ \"github.com/influxdata/REPO/fluxinit/static\"
```